### DISCUSSION TITLE
Create Encode Twitter Account

### DISCUSSION THREAD

**Discussion by omerXfaruq** [Kludex/uvicorn#1513] [Ideas]
Hello, I think it would be beneficial for encode to have an account on twitter, that way people could mention it.
[See](https://twitter.com/farukozderim/status/1534811634467741696)

cc: @tomchristie

**Comment by Kludex** [comment]
I'm willing to help, if @tomchristie thinks this makes sense.

**Reply by lovelydinosaur** [reply]
I have this account... https://twitter.com/EncodeOss which we could use.

**Reply by lovelydinosaur** [reply]
I would probably prefer not to be the primary person "behind" it, but can put in the work for helping figure out how we manage the account.

**Reply by florimondmanca** [reply]
Perhaps we could document who's got access to this account via a GitHub team, "Twitter comms" or something. Depends on if we want that account to be active at all -- people posting things such as release tweets, answering questions from people tagging us there...

I guess my point is, if this resource is to be handled by contributors, rather than strictly Encode (i.e. Tom), then we should make public note of this somewhere for accountability?

**Reply by Kludex** [reply]
You expressed concerns/suggested flow, but you didn't express your opinion @florimondmanca . Do you have one?
